packages/forms-web-app/src/dynamic-forms/dynamic-components/boolean-text/index.njk

```diff
@@ -8,18 +8,6 @@
 {% set title = question.pageTitle + " - " + journeyTitle + " - GOV.UK" %}
 {% block pageTitle %}{{ "Error: " + title if errors else title }}{% endblock %}
 
-{% if question.conditional %}
```

Comment:
we don't need this, because we have different conditional types (text, textarea, number etc) that are set during instantiation of the relevant question class